x-pack/plugins/apm/public/components/app/RumDashboard/ClientMetrics/index.tsx

```diff
@@ -22,11 +22,11 @@ const ClFlexGroup = styled(EuiFlexGroup)`
 export function ClientMetrics() {
   const { urlParams, uiFilters } = useUrlParams();
 
-  const { start, end, serviceName } = urlParams;
+  const { start, end } = urlParams;
```

Comment:
@shahzad31 I removed `serviceName` from all of these because it doesn't look like it's used and the RUM dashboard doesn't use it as a path parameter. Let me know if that's ok.

Reply:
it's actually being used, it only starts loading data once service name is selected or it's coming from url query params.

Reply:
Ok. Everywhere else in APM it's used as a path parameter (`/services/x`) but in this case it's used as a query parameter (`/csm?serviceName=x`.)

The usage here is confusing since we magically include `serviceName` in both `urlParams` and `uiFilters`. I'll make sure this works.

Reply:
Ok looks like this still works because `serviceName`, when it's a query string parameter, is included in `uiFilters`. This change doesn't break anything because `uiFilters` is already a dependency of `useFetcher`.